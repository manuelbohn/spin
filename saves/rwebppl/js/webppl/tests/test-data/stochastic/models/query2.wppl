var model = function() {
  query.add('x', flip());
  query.add('y', flip());
  return query;
};
