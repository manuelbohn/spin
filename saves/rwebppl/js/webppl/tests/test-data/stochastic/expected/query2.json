{
  "hist": {
    "{\"x\":true,\"y\":true}": 0.25,
    "{\"x\":true,\"y\":false}": 0.25,
    "{\"x\":false,\"y\":true}": 0.25,
    "{\"x\":false,\"y\":false}": 0.25
  }
}
